tests/playbooks/tests_802_1x.yml

```diff
@@ -35,15 +35,15 @@
                     - 203.0.113.2/24
                   dhcp4: "no"
                   auto6: "no"
-                802.1x:
```

Comment:
If it is not too much trouble, it would be nice to split out the 802.1x -> ieee802_1x changes into its own commit/PR.

Reply:
Have done a new PR here: #202

Will rebase in to this one if it is merged

Reply:
rebased now